backend/build.gradle

```diff
@@ -149,8 +149,8 @@ jacocoTestReport {
     dependsOn test // tests are required to run before generating the report
 }
 
-// Prefer SPRING_DATASOURCE_URL if provided
-def dbConnectionString = System.getenv("SPRING_DATASOURCE_URL") ?: "jdbc:postgresql://127.0.0.1:${System.getenv("SR_DB_PORT") ?: 5432}/simple_report"
+// Prefer SPRING_DATASOURCE_URL if provided, otherwise assume local Docker
+def dbConnectionString = System.getenv("SPRING_DATASOURCE_URL") ?: "jdbc:postgresql://db:${System.getenv("SR_DB_PORT") ?: 5432}/simple_report"
```

Comment:
this will break out ability to just run the `./gradlew` command locally outside docker, can we have it run the following logic?

- use `SPRING_DATASOURCE_URL` if set
- use `db` if running inside docker
- otherwise use localhost

Reply:
🤦 Yes it will - I was only focused on the rollback. Fix pushed.